docs/source/deploying_services_advanced.rst

```diff
@@ -0,0 +1,48 @@
+.. _deploying_services_advanced:
+
+===========================================
+How to deploy a service (developer's guide)
+===========================================
+This is a guide for developers that want to deploy Octue services themselves - it is not needed if Octue manages your
+services for you or if you are only asking questions to existing Octue services.
+
+1. Ensuring you are in the desired project, go to the `Google Cloud Run <https://console.cloud.google.com/run>`_ page
+   and create a new service
+2. Give your service a unique and useful name
+3. Choose a low-carbon region that supports Eventarc triggers and is in a convenient geographic location for you (e.g.
+   physically close to you for low latency or in a region compatible with your data protection requirements)
+3. Click "Next" and select "Continuously deploy new revisions from a source repository", then click "Set up with cloud
```

Comment:
I’d go through these and create a series of screenshots, which will get outdated over time but are a good way of anchoring the user to the steps they're taking